test/copy.jl

```diff
@@ -108,3 +108,25 @@ let x = BigFloat[1:1000;], y, z, v
     # Check that the setprecision indeed does something
     @test z != x
 end
+
+# issue #19921
+type Foo
+    a::String
+end
+
+type Bar
+    foo::Foo
+    fooDict::Dict{Foo, Int64}
+end
```

Comment:
When I do that, I get
```
Error in testset copy:
Error During Test
  Got an exception of type LoadError outside of a @test
  LoadError: error compiling anonymous: type definition not allowed inside a local scope
  while loading /home/twan/code/julia/test/copy.jl, in expression starting on line 112
ERROR: LoadError: 
Stacktrace:
 [1] (::##33#41)() at /home/twan/code/julia/test/runtests.jl:195
 [2] cd(::##33#41, ::String) at ./file.jl:69
 [3] include_from_node1(::String) at ./loading.jl:532
 [4] include(::String) at ./sysimg.jl:14
 [5] process_options(::Base.JLOptions) at ./client.jl:308
 [6] _start() at ./client.jl:374
while loading /home/twan/code/julia/test/runtests.jl, in expression starting on line 38
```
I'll rename the types to something more specific instead.